framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -835,66 +974,74 @@ private AnnotationMirror resultAnnotationHandler(
 
     public AnnotationMirror createIntValAnnotation(List<Long> intValues) {
         intValues = ValueCheckerUtils.removeDuplicates(intValues);
-        if (intValues.isEmpty() || intValues.size() > MAX_VALUES) {
+        if (intValues.isEmpty()) {
             return UNKNOWNVAL;
```

Comment:
If this was in the original code, then the original code was incorrect.  Thanks for noticing that!